modules/Sources/Generated/Resources/Localizable.strings

```diff
@@ -118,6 +118,9 @@
 "send.memo" = " memo: %@";
 "send.includeMemo" = "Want to include memo? Tap here.";
 "send.editMemo" = "Memo included. Tap to edit.";
+"send.error.insufficientFunds" = "insufficient funds";
+"send.error.invalidAmount" = "invalid amount";
+"send.error.invalidAddress" = "invalid address";
```

Comment:
Could these texts start with a big letter? Our Material design guidelines recommend that for the Android side. 
See [documentation](https://m3.material.io/components/text-fields/specs#f157ea73-f2fd-4d99-aab3-c96bd9c91eec ).

Reply:
We don't have such guidelines so normally I would leave it to designers/copywriters/anyone but me. I updated it anyway.